src/components/LoadingButton/LoadingButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  text: string;
+  'data-cy': string;
+  onDataLoad: (stopButtonLoading: () => void, goodsGroupText: string) => void;
+};
+
+export const LoadingButton: React.FC<Props> = (props) => {
+  const {
+    text,
+    'data-cy': dataCy,
```

Comment:
as for me - it's a bit strange to pass this prop written in kebab-case to convert it to camelCase to pass it to kebab-cased attribute
we can pass this prop camelCased to this component and avoid not needed converting